packages/tooling/fast-tooling-react/src/form/templates/template.control.utilities.tsx

```diff
@@ -89,6 +90,7 @@ abstract class ControlTemplateUtilities<P, S> extends React.Component<
                     className={className}
                     disabled={this.props.disabled}
                     onChange={this.handleSetDefaultValue}
```

Comment:
More of an FYI (unless you feel like integrating it): [React Context](https://reactjs.org/docs/context.html) is a really good fit for this problem. It would allow us to not pass strings directly from component to component, and instead just broadcast them from the root component, where any interested component could opt-in without wiring props to it.

Reply:
Good to know. Maybe something to think about as part of a larger refactor. I think the property dependency chain could be simplified quite a bit.